cmd/cosign/cli/verify/verify_blob_test.go

```diff
@@ -959,6 +957,91 @@ func TestVerifyBlobCmdWithBundle(t *testing.T) {
 	})
 }
 
+func TestVerifyBlobCmdInvalidRootCA(t *testing.T) {
+	keyless := newKeylessStack(t)
+	// Change the keyless stack.
+	_ = newKeylessStack(t)
+	t.Run("Invalid certificate root explicit certRef", func(t *testing.T) {
+		// Change the keyless stack.
+		_ = newKeylessStack(t)
```

Comment:
Don't you do that outside the test? (couple lines up)

Reply:
Fixed in latest commit